snap/snapcraft.yaml

```diff
@@ -163,13 +89,12 @@ parts:
     - on armhf: [qemu-system-arm]
     - on arm64: [qemu-system-arm]
     - qemu-utils
-    - libslang2
```

Comment:
I've had to re-introduce this dependency when I merged this with my strict confinement stuff, qemu needs it:

Feb 06 13:56:09 box2 multipassd[19512]: /snap/multipass/x2/usr/bin/qemu-system-x86_64: error while loading shared libraries: libslang.so.2: cannot open shared object file: No such file or directory

Why did you remove?

Reply:
`snapcraft` didn't complain about it, but I'll bring it back then.

Reply:
Just a note, `libslang2` was originally added to fix the snap on Solus Linux.